docs/labs/15-Figma.md

```diff
@@ -0,0 +1,330 @@
+# Figma Lab
+
+- [Overview](#overview)
+- [Creating Your Figma Account](#creating-your-figma-account)
+- [Creating Your First Figma Project](#creating-your-first-figma-project)
+- [Adding Elements to Project](#adding-elements-to-project)
+    - [Adding a Rectangle](#adding-a-rectangle)
+    - [Adding a Circle](#adding-a-circle)
+    - [Adding a Label](#adding-a-label)
+- [Adjusting Properties](#adjusting-properties)
+    - [Changing Corners](#changing-corners)
+    - [Changing The Label](#changing-the-label)
+    - [Changing Font](#changing-font)
+    - [Changing Colors](#changing-shape-colors)
+- [Challenge Object 1](#challenge-object-1)
+- [Challenge Object 2](#challenge-object-2)
+- [Porting Objects to a ACM Java Project](#porting-objects-to-a-acm-java-project)
+    - [Exporting as attributes](#exporting-as-attributes)
+    - [Exporting as an Image](#exporting-as-an-image)
+- [Drawbacks](#drawbacks)
+    - [Gradients](#gradients)
+    - [Mixed Colors on Labels](#mixed-colors-on-labels)
+    - [And More..](#and-more)
+- [What Next..](#what-next)
+
+## Overview
+
+Have you heard of Figma before this class? If not, this lab is for you.
+
+This lab will be teaching you the simplest core functions of figma to
+help you create a seamless, connected UI across all screens in your Comp 55 Projects.
+
+Figma is a powerful tool to help you line up your ideas before you
+begin coding them, which is important if you generally like
+to jump in and begin programming.
+
+Upon completing this lab you will know how to:
+
+- Create a figma project
+- Add objects onto your figma project
+- Edit object's properties
+- Be able to port the design into a java project
+
+Submission for this lab will be:
+// TODO am I even allowed to add submissions in the first place?
```

Comment:
I think it would be easy enough to have them come up with or to submit the UI design for a personal about me application, almost like a webpage but as if it were in ACM or something.  An assignment that allows for some customization even if it is contrived.